docs/one-click-integrations/github-integration.md

```diff
@@ -139,6 +139,58 @@ The table below lists the data that the GitHub integration collects from your Gi
 
 <sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on GitHub only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on GitHub will continue to show in Pulse.
 
+## Which permissions does Pulse need from GitHub? {: id="gh-permissions"}
+
+Pulse requests only the necessary permissions from GitHub to collect your organization changes and deployment data and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
```

Comment:
The current version seems to indicate that Pulse collects two types of data:

- Organization changes
- Deployment data

Something along these lines would be more focused on the main data that Pulse collects to calculate the metrics, which are changes and deployments:

```suggestion
Pulse requests only the necessary permissions from GitHub to collect changes and deployment data from the repositories in your organization and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
```

I don't think we need to mention here all the details that are listed in the table (which would include things like the members and teams in the organization, for example).